website-next/components/common/env-vars.ts

```diff
@@ -29,6 +25,39 @@ export const IS_TEST_ENVIRONMENT: boolean =
 export const IS_BROWSER_TEST_DEBUG: boolean =
   IS_TEST_ENVIRONMENT && process.env.REACT_APP_BROWSER_TEST_DEBUG === 'true'
 
+export const DEVELOPMENT_ENV: boolean =
+  !PRODUCTION_OR_STAGING_CONFIG && !IS_TEST_ENVIRONMENT && HOSTNAME === 'localhost'
+
+type BackendType =
+  | 'bff' // proxied calls via the Remix BFF
+  | 'direct' // direct calls to the backend
+
+const LOCAL_BACKEND_PORTS: { [type in BackendType]: number } = {
+  direct: 8001,
+  bff: 8002,
+}
+
+export const BACKEND_TYPE: BackendType = DEVELOPMENT_ENV ? 'bff' : 'direct'
+
+export function isBackendBFF(): boolean {
+  return BACKEND_TYPE === 'bff'
+}
+
+export const UTOPIA_BACKEND =
+  (PRODUCTION_OR_STAGING_CONFIG
+    ? BASE_URL
```

Comment:
is this right? IIUC the original code was `BASE_URL + 'v1/'`

Reply:
yep, in this case it's the same (`BASE_URL + v1/`) on everything except localhost, where it's `scheme://<host>:<port>/ + v1/`